README.md

```diff
@@ -80,6 +80,29 @@ If you need a numeric version even for rawhide:
 ```
 
 
+## Caching
+
+To avoid a single point of failure on Bodhi API, we can cache successfully
+fetched data, and use them when there is an issue.
+
+```python
+>>> aliases = get_distro_aliases(cache=True)
+>>> [x.namever for x in aliases["fedora-all"]]
+['fedora-39', 'fedora-40', 'fedora-rawhide']
+```
+
+To configure parameters for the cache, or use your own caching mechanism, pass
+an object instead of `True`.
```

Comment:
I would probably add an example of what is expected to appear…

1. _your own caching mechanism_ implies an object satisfying some protocol/interface, so that's clear
2. _configure parameters_ is not that straightforward though, is it a dictionary, dataclass, or something else?

Reply:
Thank you @mfocko, good point. Hopefully fixed now.